test/unit/pusher_test.rb

```diff
@@ -258,7 +258,7 @@ class PusherTest < ActiveSupport::TestCase
         should "be false if not owned by user and an indexed version exists" do
           create(:version, rubygem: @rubygem, number: '0.1.1')
           refute @cutter.authorize
-          assert_equal "You do not have permission to push to this gem.", @cutter.message
+          assert_includes @cutter.message, "You do not have permission to push to this gem."
```

Comment:
@simi Alright, I've made the change. Now the integration test checks for less details (which makes sense for a high-level test) and the unit test instead covers the exact message. Thanks for the feedback!